commands/version.go

```diff
@@ -54,6 +115,50 @@ func printFiglet() {
 	fmt.Printf(figletColoured)
 }
 
+func getGatewayDetails(m map[string]interface{}) (version, sha, commit string) {
+	v := m["version"].(map[string]interface{})
```

Comment:
Any struct we can reference/vendor from faas repo?

Reply:
we could vendor in the `types` package from `gateway` to use the `VersionInfo`, if you are okay with that?

Reply:
yes apply "prune" if it's not there yet